src/test/resources/android-app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
New Android plugin need a AndroidManifest file to be applied on a projet. If you have any better idea I am interested to hear it